extensions/functions_datetime.yaml

```diff
@@ -307,6 +307,12 @@ scalar_functions:
           - name: y
             value: interval_day<P>
         return: timestamp
+      - args:
+          - name: x
+            value: date
+          - name: y
+            value: interval_compound<P>
```

Comment:
All of the other variants only add day components or higher to the date.  This also has subday components so I'm not sure it's appropriate.